Formula/sqlite_fdw.rb

```diff
@@ -0,0 +1,66 @@
+class SqliteFdw < Formula
+  desc "SQLite Foreign Data Wrapper for PostgreSQL"
+  homepage "https://github.com/pgspider/sqlite_fdw/"
+  url "https://github.com/pgspider/sqlite_fdw/archive/v1.3.0.tar.gz"
+  sha256 "61283b7f33b2a727af722fcfcdfe3cb7d6847e1f43e12a4b3554ab7190bfe9b3"
+  license "MIT"
+
+  depends_on "postgresql"
+  depends_on "sqlite"
+
+  def install
+    mkdir "stage"
+    system "make", "install", "USE_PGXS=1", "DESTDIR=#{buildpath}/stage"
+    lib.install Dir["stage/#{HOMEBREW_PREFIX}/lib/*"]
+    share.install Dir["stage/#{HOMEBREW_PREFIX}/share/*"]
+  end
+
+  test do
```

Comment:
Is there a smaller test that could be done?

Reply:
I'm not sure how. `sqlite_fdw` is a Postgresql extension, so in order to properly test it, we need to start up Postgresql and verify that it can load the extension and use it. Do you have another suggestion?